inductiva/molecules/scenarios/utils.py

```diff
@@ -1,15 +1,27 @@
 """Tools to analyze molecular dynamics simulations."""
-import MDAnalysis as mda
-from MDAnalysis import transformations
-from MDAnalysis.analysis import align
+try:
+    import MDAnalysis as mda
+    from MDAnalysis import transformations
+    from MDAnalysis.analysis import align
+except ImportError:
+    mda = None
+    transformations = None
+    align = None
+
+
+def _check_mdanalysis() -> None:
+    if mda is None or transformations is None or align is None:
+        raise ImportError("MDAnalysis is not installed.")
```

Comment:
We need to tell the user how to install it. Either we are assuming that by saying this, the user will be doing "pip install MDAnalysis" or if we want the user to do pip install inductiva[molecules_viz] then we need to tell it explicitly in this error message.

Reply:
I think messages need to be as helpful as possible. We should be explicit.